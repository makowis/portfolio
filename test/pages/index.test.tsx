import React from 'react';
import { render } from '@testing-library/react';
import Home from '../../app/page';

it('renders correctly', () => {
  const { container } = render(<Home />);
  expect(container.childNodes).toMatchSnapshot();
});
