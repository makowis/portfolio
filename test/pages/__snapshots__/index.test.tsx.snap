// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <main>
    <h1>
      makowis.dev
    </h1>
    <p>
      ポートフォリオサイトという名のリンク集
    </p>
    <section>
      <h2>
        Links
      </h2>
      <ul>
        <li>
          <a
            href="https://twitter.com/mako_wis"
          >
            Twitter
          </a>
        </li>
        <li>
          <a
            href="https://www.facebook.com/makoto.henmi"
          >
            Facebook
          </a>
        </li>
        <li>
          <a
            href="https://www.instagram.com/mako_wis/"
          >
            Instagram
          </a>
        </li>
        <li>
          <a
            href="https://github.com/makowis"
          >
            GitHub
          </a>
        </li>
        <li>
          <a
            href="https://speakerdeck.com/makowis"
          >
            Speaker Deck
          </a>
        </li>
        <li>
          <a
            href="https://mako-wis.hatenablog.com/"
          >
            Blog
          </a>
        </li>
      </ul>
    </section>
  </main>
  <footer>
    <a
      href="https://github.com/makowis"
      rel="noopener noreferrer"
      target="_blank"
    >
      Powered by mako_wis
    </a>
  </footer>
</div>
`;
